### PR TITLE
feat: cartlist management page

### PR DESCRIPTION
**Pull request by NandhiniKrishnamoorthy**
Added Cart list management page, Please Review

image

image

### CODE REVIEW

File: Cart_management/cartlist.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <!-- font awesome icons -->
```
Comment (rahul31agrawal) [inline Cart_management/cartlist.html:9]: Try to remove commented out codes before pushing it to github.

**Review by rahul31agrawal — APPROVED** [review]
Looks good to merge

File: Cart_management/cartlist.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    <!-- favicon -->
+    <link rel="icon" href="https://img.icons8.com/flat-round/64/000000/crown--v1.png"/>
+    
+    <!-- style 10 component library -->
+    <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/padding.css" />
+    <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/margin.css" />
+    <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/typography.css" />
+    <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/card.css" />
+    <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/ratings.css" />
+
```
Comment (ranitasahaa1230) [inline Cart_management/cartlist.html:23]: You can put all these in a CSS file and access the link from there

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to merge

File: Cart_management/cartlist.html (hunk 2 above)
Comment (NandhiniKrishnamoorthy) [reply Cart_management/cartlist.html:23]: Will Update

**Pull request merged**